std/math.d

```diff
@@ -2751,8 +2751,7 @@ if (isFloatingPoint!T)
     int exp;
     real mantissa = frexp(123.456L, exp);
 
-    // check if values are equal to 19 decimal digits of precision
-    assert(equalsDigit(mantissa * pow(2.0L, cast(real) exp), 123.456L, 19));
+    assert(approxEqual(mantissa * pow(2.0L, cast(real) exp), 123.456L));
```

Comment:
> If equalsDigit and approxEqual are sufficiently the same, then there's no problem. :)

`approxEqual` (used for the public examples) is much weaker, than `equalsDigit`. `approxEqual` checks by a difference of `1e-5` by default, whereas the `equalsDigit` tests here are by a precision of `1e-19`.